src/_locales/en/messages.json

```diff
@@ -221,7 +251,7 @@
     "description": "This is included in code-style in optionHelperTextAutocorrectionEmoji."
   },
   "optionHelperTextAutocorrectionEmojiLink": {
-    "message": "https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/",
+    "message": "https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-options-page&utm_campaign=unicodify-options-page-inline-link",
```

Comment:
I think this can be removed now. Also, it looks like you are missing some German translations above.

Reply:
Naw, let’s not remove this, it allows to see statistics about where users came from when the add-on is downloaded. See https://extensionworkshop.com/documentation/manage/monitoring-extension-usage-statistics/

Reply:
Interesting. I was referring the the whole URL/message now that we have added support for Thunderbird and Chrome. It was already removed from the German localization file above.

Reply:
Oh yeah, you’re right. I was also considering adding entries for all URLs so they are localizable, but they redirect anyway, so IMHO this should not be a big problem. :slightly_smiling_face:

Reply:
This use of it also needs to be removed from the options page: https://github.com/rugk/unicodify/blob/34b79d14d6e9b14b66bb0c5f77e696fff0d9b43b/src/options/options.html#L64